src/crimson/os/seastore/segment_cleaner.h

```diff
@@ -503,6 +505,18 @@ class SegmentCleaner : public JournalSegmentProvider {
     start();
   }
 
+  store_statfs_t stat() const {
+    store_statfs_t st;
+    st.total = get_total_bytes();
+    st.available = get_total_bytes() - get_used_bytes();
+    st.allocated = get_used_bytes();
```

Comment:
does allocated mean used bytes?  what the difference between allocated and data_stored?

Reply:
Internally, get_used_bytes returns the sum of all live extents, which is generally somewhat larger than the amount of data stored by the user.  This should match the allocated field of store_statfs_t.  We can refine this later if needed.